### PR TITLE
set cursor: pointer on document element when on ios

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes https://github.com/ampproject/amphtml/issues/2054

### CODE REVIEW

**Comment by erwinmombay** [comment]
@dvoytenko PTAL

File: test/functional/test-styles.js [hunk 1]
```diff
@@ -41,4 +44,12 @@ describe('Styles', () => {
     expect(getStyle(document.body, 'visibility')).to.equal('visible');
     expect(getStyle(document.body, 'animation')).to.equal('none');
   });
+
+  it('should set cursor:pointer on document element correctly', () => {
+    const elem = document.documentElement;
+    expect(elem.style.cursor).to.not.be.ok;
+    sandbox.stub(platform, 'isIos').returns(true);
+    styles.installStyles(document, '', () => {});
+    expect(elem.style.cursor).to.be.ok;
```
Comment (dvoytenko) [inline test/functional/test-styles.js]: Let's do `to.equal.pointer()` - we want to be very explicit here.

File: test/functional/test-styles.js [hunk 2]
```diff
@@ -41,4 +44,12 @@ describe('Styles', () => {
     expect(getStyle(document.body, 'visibility')).to.equal('visible');
     expect(getStyle(document.body, 'animation')).to.equal('none');
   });
+
+  it('should set cursor:pointer on document element correctly', () => {
```
Comment (dvoytenko) [inline test/functional/test-styles.js]: "should set cursor:pointer on iOS ...."

**Comment by dvoytenko** [comment]
LGTM with two comments.

File: test/functional/test-styles.js (hunk 1 above)
Comment (erwinmombay) [inline test/functional/test-styles.js]: good point, i think i just forgot to switch it.

Comment (erwinmombay) [inline test/functional/test-styles.js]: done.

File: test/functional/test-styles.js (hunk 2 above)
Comment (erwinmombay) [inline test/functional/test-styles.js]: done.

**Pull request merged**